app/validators/open_before_closed_validator.rb

```diff
@@ -6,8 +6,8 @@ def validate_each(record, attribute, _value)
 private
 
   def before_closed?(record)
-    opened = record.try(:opened_date) || record.try(:oim_project_opened_date)
-    closed = record.try(:closed_date) || record.try(:oim_project_closed_date)
+    opened = record.try(:opened_date) || record.try(:oim_project_opened_date) || record.try(:disease_case_opened_date)
+    closed = record.try(:closed_date) || record.try(:oim_project_closed_date) || record.try(:disease_case_closed_date)
```

Comment:
Would there be any drawback to re-using `opened_date` and `closed_date` metadata fields on this new document type? 🤔 Or do they need to have a unique name for each document type?

Reply:
Yes I thought the same thing (hence the legacy opened and closed dates in the other file 🙈), I think it does need to be different since the public facing page wouldn't load when it was opened and closed dated so I think somewhere it makes it unhappy... which is a shame since all the validations etc would be the same